solutions/swb-reference/src/templates/README.md

```diff
@@ -1,30 +1,48 @@
-# As we enable more hosting account templates, we will document the purpose of each here
+# Onboarding a hosting account template
 
-## onboard-account.cfn.yaml
-A CFN template that creates all resources SWB needs to manage the relationship between the main and hosting accounts.
+Here are a few onboarding template choices that you could use for onboarding your hosting account. The difference between each of these templates is based on the level of network flexibility you need. This is where your SWB compute resources would be hosted.
 
-## onboard-account-byon.cfn.yaml
-A CFN template that does everything onboard-account.cfn.yaml does, except allows the hosting account to use its own VPCs.
+----
 
-## The following are CFN templates that help setting up networks for transit gate ways in the BYON scenario
-### new-firewall-vpc-TGW.yaml
-A template to help a Network account set up VPCs, a firewall, and a Transit Gateway. 
+## Onboard Account
+This is the default [template]((./onboard-account.cfn.yaml)) for onboarding a hosting account. This CFN template creates all resources SWB needs to manage the relationship between the main and hosting accounts. The network components in this template are associated with an Internet Gateway, therefore workspaces in this account network will have access to internet.
 
-This template was originally designed for use with Palo Alto Networks' VM Series firewall offering on AWS Marketplace. However, the template can be modified to leverage any firewall offering on AWS Marektplace that has an AMI.
+----
 
-### onboard-account-tgw.cfn.yaml
-A template to help hosting/member accounts integrate with the Network account. 
+## Onboard Account (Bring Your Own Network)
+This [template]((./onboard-account-byon.cfn.yaml)) does everything onboard-account.cfn.yaml does, except allows the user to bring their own network components (VPC and subnets). This template therefore gives the user more granular control by using pre-configured networks for hosting workspaces. These pre-confurations can include, but are not limited to, network traffic monitoring, cross account network control (using route tables, [VPC peering](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/peer-with-vpc-in-another-account.html), [PrivateLink](https://docs.aws.amazon.com/vpc/latest/userguide/endpoint-services-overview.html) etc.), and additional security layers.
```

Comment:
confurations --> configurations typo

Reply:
I'd relpace `onboard-account.cfn.yaml` with a link to the `Onboard Account` section above